yarn-project/acir-simulator/src/avm/journal/journal.ts

```diff
@@ -136,7 +130,7 @@ export class AvmJournal {
     const incomingFlush = this.flush();
```

Comment:
Why do we need this here? the object returned by `this.flush()` seems to be very similar to `this`. In particular, it's not a deep copy IIUC?

Reply:
This is a good point